### PR TITLE
DOCS-6553: Updates to the resync procedure

### PR DESCRIPTION
**Pull request by kallimachos**
This change is [Reviewable](https://reviewable.io/reviews/mongodb/docs/3002)

### CODE REVIEW

File: source/tutorial/resync-replica-set-member.txt [hunk 1]
```diff
@@ -48,40 +48,43 @@ Procedures
...
+   :setting:`~storage.dbPath` directory.
+
+This procedure relies on MongoDB's regular process for
+:doc:`Replica Set Syncing </core/replica-set-sync>`. This stores the current
+data on the member. For an overview of MongoDB initial sync process, see the
 :doc:`Replica Set Syncing </core/replica-set-sync>` section.
 
-If the instance has no data, you can simply follow the
+Initial sync operations can impact the other members of the set and
+create additional traffic to the primary. They require another member of the
```
Comment (kay-kim) [inline source/tutorial/resync-replica-set-member.txt]: just one nit -- should we clarify what "They" refer to ?

Comment (kallimachos) [reply source/tutorial/resync-replica-set-member.txt]: Good idea, I'll clarify.

Comment (kallimachos) [reply source/tutorial/resync-replica-set-member.txt]: Done!

**Comment by kay-kim** [comment]
merged!  thank you!

**Pull request closed without merging**